marc_export/src/main/java/whelk/export/marc/MarcCliExport.java

```diff
@@ -56,6 +59,13 @@ public static void main(String[] args)
         if (args.length != 2 && args.length != 1)
             printUsageAndExit();
 
+        if (args[0].equals("--sao"))
```

Comment:
We should probably provide a filter on inScheme instead of hardcoding this.

Reply:
I disagree. This is for MARC export for a very specific legacy-purpose, and we should not be building new complex generic structures for other "future" sorts of MARC exports.

Reply:
I figured that was the reason. If we can guarantee that noone wants terms from any other scheme, that is fair. (We already have complex, hardcoded specific constructs for all sorts of MARC exports. Just note that `sao` is way more domain specific than `bib`, `auth` or `hold`.)